### PR TITLE
Added documentation of .znn file type.

### PR DESCRIPTION
**Pull request by william-silversmith**
You'll probably want to double check the references I cited and the description of the filter initializations to ensure they're correct.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -8,11 +8,14 @@ the python build instructions as it will result in the interface and a compiled
...
 
-Acquiring an Amazon Machine Image (AMI) -- Recommended
+Acquiring a Machine Image -- Recommended
 ------------------------------------------------------
 
-We have some Amazon Machine Images set up and read to go to train on AWS. 
-This is the easiest method. 
+We have some machine images set up and ready to go for training on:
+1. `Amazon Web Services <aws.amazon.com>`_ (called AMIs)
+2. `Google Cloud Platform <cloud.google.com>`_ (called Snapshots)
```
Comment (jingpengwu) [inline docs/install.rst]: AWS and Google both have AMI and snapshots.

File: docs/install.rst [hunk 2]
```diff
@@ -8,11 +8,14 @@ the python build instructions as it will result in the interface and a compiled
...
+
+This is the easiest method as the program's dependencies are loaded and the compilation is fully optimized.
```
Comment (jingpengwu) [inline docs/install.rst]: we may not be able to share the AMI with MKL with others due to the license can only be used in our group.

File: docs/install.rst (hunk 1 above)
Comment (william-silversmith) [inline docs/install.rst]: Hmm, I thought AMI stood for Amazon Machine Image? Are snapshots different?

File: docs/install.rst (hunk 2 above)
Comment (william-silversmith) [inline docs/install.rst]: Good point! I'll adjust the language.

**Pull request merged**